openy.packages.yml

```diff
@@ -168,3 +193,90 @@ locations:
     - location_finder
     - openy_prgf_loc_finder
     - openy_loc_branch
+
+locations_ext:
+  name: 'Locations (extended) Package'
+  help: '<p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.</p>
```

Comment:
Let's avoid of Lorem Ipsum, please use real content from real Y sites

Reply:
the real description will be added in the next task since they are not approved yet. Please skip for now.